source/client/styles/global.css

```diff
@@ -157,13 +157,21 @@ input[type="number"] {
   padding: 0.75rem 1.75rem;
   display: inline-block;
   transition: var(--default-transform-transition), opacity 350ms ease-in-out;
+  -webkit-transition: var(--default-transform-transition), opacity 350ms ease-in-out;
+  -moz-transition: var(--default-transform-transition), opacity 350ms ease-in-out;
+  -ms-transition: var(--default-transform-transition), opacity 350ms ease-in-out;
+  -o-transition: var(--default-transform-transition), opacity 350ms ease-in-out;
 }
 
 .primary-button:not(:disabled):hover,
 .secondary-button:not(:disabled):hover,
 .tertiary-button:not(:disabled):hover,
 .destructive-button:not(:disabled):hover {
   transform: var(--default-scale);
+  -webkit-transform: var(--default-scale);
+  -moz-transform: var(--default-scale);
+  -ms-transform: var(--default-scale);
+  -o-transform: var(--default-scale);
```

Comment:
Why????

Reply:
I have no clue tbh, I think it might have been part of a merge? see commit 5411c4c, authored during the 3pm today